openlibrary/plugins/openlibrary/js/ile/utils/SelectionManager/SelectionManager.js

```diff
@@ -85,6 +85,7 @@ export default class SelectionManager {
     toggleSelected(clickEvent) {
         // If there is text selection or the click is on a link that isn't a select handle, don't do anything
         if (window.getSelection()?.toString() !== '' ||
+            $(clickEvent.target).closest('.searchResultItemCTA').length > 0 ||
             ($(clickEvent.target).closest('a').is('a') &&
```

Comment:
We can make this a little tighter :) This will also handle any other `<button>` elements.
```suggestion
            ($(clickEvent.target).closest('a,button').length > 0 &&
```

Reply:
looks good to me. tested and it is working properly

Reply:
the read and want to read buttons are not button elements.  but still, this solves the issue. @scottbarnes could you look into this?